ServiceWebsite/ServiceWebsite/ClientApp/src/app/modules/individual-journey/pages/about-your-computer/about-your-computer.component.spec.ts

```diff
@@ -1,8 +1,15 @@
 import { AboutYourComputerComponent } from './about-your-computer.component';
 import { CanCreateComponent } from '../individual-base-component/component-test-bed.spec';
+import { CannotProceeedUntilChoiceIsSelected } from '../../components/suitability-choice-page-base.component.spec';
 
 describe('AboutYourComputerComponent', () => {
   it('can be created', () => {
     CanCreateComponent(AboutYourComputerComponent);
   });
+  it('cannot proceed to next step until pressing choice, after submit value is bound', () => {
+    const fixture = CannotProceeedUntilChoiceIsSelected(AboutYourComputerComponent);
+
+    // and value is bound
+    expect(fixture.component.model.camera).toBe(0);
```

Comment:
I'm guessing this zero maps up to the first answer, the choice-no. I think it might be easier to read the expectation if it's written like:
```typescript
const modelValue = HasAccessToCamera[fixture.component.model.camera];
expect(modelValue).toBe(HasAccessToCamera.No);
```